backend/src/journey/entity/transportation.entity.ts

```diff
@@ -0,0 +1,53 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { Trip } from "./trip.entity";
+
+
+export class Transportation {
+
+  @ApiProperty({
+    description: "Name of transportation vehicle.",
+    type: String,
+    required: true
+  })
+  name?: string;
+
+  @ApiProperty({
+    description: "Trips depending on this transportation.",
+    type: [Trip],
+    required: true
+  })
+  trips!: Trip[];
```

Comment:
Is this information not already contained in `leg`?

Reply:
The trip information is about the transportation itself. So it's not about the leg of a journey itself but about the transportation that is used by the leg.

Reply:
Kurz mal auf deutsch: Also sind Trips Fahrten die ein Fahrzeug (oder eine Line) macht?

Reply:
> Kurz mal auf deutsch: Also sind Trips Fahrten die ein Fahrzeug (oder eine Line) macht?

Genau, das gibt an, was das Fahrzeug/Transportmittel aus einem Leg so tut.

Reply:
Okay dann eine Idee für den Kommentar: 

> Trips that are executed by this transportation.

Transportation hängt dann aber nicht von Leg ab.